oeps/best-practices/oep-0051-bp-conventional-commits.rst

```diff
@@ -148,6 +150,8 @@ Choosing the commit type label to use for a commit is important, but it's only t
 
 **feat vs fix**: some user-visible changes to features could be classified as "feat" or "fix".  Choose "feat" if the change adds to the set of features.  Choose "fix" if the change affects how a feature behaves.  Yes, this is still subjective.
 
+**squash vs temp or fix**: commits that are intended to be squashed before merge should use the "squash" prefix. Although "temp" or "fix" may feel appropriate, they are not meant to be used in this way and convey something meaningful after a PR merges, where "squash" commits are not intended to be merged.
```

Comment:
You might also draw the distinction that `temp:` is intended to mean that the _change_ is temporary, whereas `squash:` means that the _commit_ is temporary but the changes it contains likely are not and will be incorporated into a larger commit with a different semantic type.

Reply:
I like this language and updated, but made additional adjustments.

Reply:
I like it!  Resolved.